pkg/api/networkservice/connection.pb.go

```diff
@@ -46,19 +46,22 @@ const (
 type State int32
 
 const (
-	State_UP   State = 0
-	State_DOWN State = 1
+	State_UP                State = 0
+	State_DOWN              State = 1
+	State_REFRESH_REQUESTED State = 2
 )
 
 // Enum value maps for State.
 var (
 	State_name = map[int32]string{
 		0: "UP",
 		1: "DOWN",
+		2: "REFRESH_REQUESTED",
 	}
 	State_value = map[string]int32{
-		"UP":   0,
-		"DOWN": 1,
+		"UP":                0,
+		"DOWN":              1,
+		"REFRESH_REQUESTED": 2,
```

Comment:
Hmm.. What about that?

```suggestion
		"REFRESHING": 2,
```

Reply:
@edwarnicke 
What do you think?

Reply:
I think it really is 'REFRESH_REQUESTED' ... 'REFRESHING' implies that the refresh is actively happening now... which it isn't yet, until the NSC sends a refresh 'Request'.